pkg/k8s/pipelineRun.go

```diff
@@ -43,11 +45,16 @@ type PipelineRun interface {
 }
 
 type pipelineRun struct {
-	client stewardv1alpha1.PipelineRunInterface
-	apiObj *api.PipelineRun
-	copied bool
+	client         stewardv1alpha1.PipelineRunInterface
+	apiObj         *api.PipelineRun
+	copied         bool
+	changes        []func(*api.PipelineStatus) (commitRecorderFunc, error)
+	commitRecorder []commitRecorderFunc
```

Comment:
```suggestion
	commitRecorders []commitRecorderFunc
```
As this is a list please use plural

Reply:
done